docs/source/dev/debugging.rst

```diff
@@ -17,6 +17,7 @@ The following build options can assist the investigation:
 * ``PMACC_VERBOSE=<N>`` sets log detail level for pmacc, highest level is 127.
 * ``PMACC_BLOCKING_KERNEL=ON`` makes each kernel invocation blocking, which helps to narrow a crash down to a particular kernel.
 * ``CUPLA_STREAM_ASYNC_ENABLE=OFF`` disables asynchronous streams, also helps to narrow a crash down to a particular place.
+* ``CMAKE_BUILD_TYPE=Debug`` compile in debug mode where all assertions are activated.
```

Comment:
I think we should also add a warning that on GPUs it should be avoided as possible, and only used as last resort ?

Reply:
I added a note that kernels would be slower. In general, it is useful to test new code in debug mode too.